src/post_section/post_section.jsx

```diff
@@ -1,48 +1,50 @@
 import React from 'react';
 import './post_section.css';
+const linksForFirstTable = [
+    {
+        name: "facebook",
+        src: "/photos/facebook.png",
+    },
+    {
+        name: "twitter",
+        src: "/photos/twitter.png",
+    },
+    {
+        name: "phone abyss",
+        src: "/photos/phone.png",
+    },
+];
+const linksForSecondTable = [
+    {
+        name: "facebook2",
+        src: "/photos/facebook.png",
+        text: "Share on Facebook"
+    },
+    {
+        name: "twitter2",
+        src: "/photos/twitter.png",
+        text: "Share on Twitter"
```

Comment:
just noticed. why do you need 2 arrays for this? they look the same, the only difference is that the second array has `text` in objects. you can use  single array and just not render `text` where it's not needed. so remove `linksForFirstTable` and figure out how to render everything using just `linksForSecondTable`